### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by thaotonto**

### CODE REVIEW

**Comment by ducbet** [comment]
ok

**Comment by noobsi** [comment]
ok

File: app/views/admin/movies/_form.html.erb [hunk 1]
```diff
@@ -29,4 +29,8 @@
   <%= f.date_field :release_date, class: "form-control", min: Date.today %>
 
   <%= f.submit yield(:title), class: "btn btn-primary" %>
+
+   <span class="picture">
```
Comment (DuongNT1512) [inline app/views/admin/movies/_form.html.erb]: indent

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,35 +1,61 @@
...
+</div>
 
-</header>
+<div class="header" >
+  <div class="main_nav">
+    <div class="nav fl">
+      <ul>
+        <li><%= link_to t(".home"), root_path %></li>
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown">
```
Comment (DuongNT1512) [inline app/views/layouts/_header.html.erb]: dunfg link_to nhé em

File: config/initializers/carrier_wave.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+if Rails.env.production?
+  CarrierWave.configure do |config|
+    config.fog_credentials = {
+      # Configuration for Amazon S3
+      :provider              => "AWS",
```
Comment (DuongNT1512) [inline config/initializers/carrier_wave.rb]: convention

**Pull request merged**